chain/epoch-manager/src/adapter.rs

```diff
@@ -292,6 +292,18 @@ pub trait EpochManagerAdapter: Send + Sync {
         )))
     }
 
+    /// This function is only needed to have a 'return true' implementation
+    /// as part of MockEpochManager to make tests work with stateless validation
+    /// enabled.
+    /// TODO(#10640): remove this after we get rid of MockEpochManager
+    fn is_chunk_producer_for_epoch(
```

Comment:
Yeah, I see how it can be confusing. What I meant here is that it doesn't have to be a part of `EpochManagerAdapter` trait. I guess it is not that important, so I will just drop the comment and close the issue.